Scripts/ClientEvents.cs

```diff
@@ -2,100 +2,78 @@
 
 namespace CharismaSDK.Events
 {
-    /// <summary>
-    /// Options with which to start the interaction with Charisma.
-    /// </summary>
-    public class StartEvent
+    public abstract class ClientEvent

```

Comment:
nice, good structure change. not sure if we'd use that abstract base class in our implementation since Colyseus commands take in objects directly, but its clean to have